### DISCUSSION TITLE
Welcome to FarNet Discussions!

### DISCUSSION THREAD

**Discussion by nightroman** [nightroman/FarNet#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas.
  * Ask questions.
  * Engage with other community members.

**Comment by Usurer** [comment]
Hi guys.
I'm sorry if I'm here with a stupid nubie question, but my experience with unmanaged code is like absolute zero and I have some tough time trying to figure out, how do FarNet modules get themselves into the Far. I'd really appreciate someone showing me the right direction, some RTFMs to read or just keywords to google.

**Reply by nightroman** [reply]
Hi @Usurer
Welcome to FarNet!

Ideally you should have created a separate discussion for this question.
Use the button "New Discussion", set category "Q&A", title, question.
But it's fine to ask here as well.

What do you mean by "how do FarNet modules get themselves into the Far"?

If it is how to install existing modules, then start with reading the [README](https://github.com/nightroman/FarNet#readme).
If it is how to build your own modules, then start with reading [wiki/FarNet](https://github.com/nightroman/FarNet/wiki/FarNet).

**Reply by Usurer** [reply]
Thanks for reply, @nightroman
Sorry for starting the discussion in the wrong thread.
I've recreated it at https://github.com/nightroman/FarNet/discussions/39 to keep things in proper order. And I've tried to be more precise in describing what exactly I don't understand )
